lib/src/table/DropdownTheme.jsx

```diff
@@ -1,14 +1,16 @@
 import { componentTokens } from "../common/variables";
 export default {
   dropdown: {
-    // Breadcrumbs tokens
+    // ActionIcon size tokens
     buttonIconSize: "16px",
```

Comment:
Some of this tokens are different from the dropdown component tokens. Which is the reason of this difference? Do we have a design for this?
In comments, we have the dropdown componentTokens values:
![image](https://github.com/dxc-technology/halstack-react/assets/10975289/03e5c0a2-e843-4c99-883b-fe3a5f93ef1d)